prefetch.bs

```diff
@@ -863,6 +863,16 @@ The \`<code><dfn http-header>Sec-Speculation-Tags</dfn></code>\` HTTP request he
 
 The header field is an [[RFC9651]] Structured Header whose value must be a [=structured header/List=]. The list can contain either [=structured header/Token=] or [=structured header/String=] values. String values represent developer-provided tags, whereas token values represent predefined tags. As of now, the only predefined tag is `null`, which indicates a speculative navigation request with no developer-defined tag.
 
+<h2 id="interaction-with-other-specs">Interaction with other specifications and concepts</h2>
+
+<div algorithm="clear prefetch cache for origin patch">
+  Modify [$clear cache for origin$] to add the following step at the end.
+
+  1. [=list/For each=] [=prefetch record=] |prefetchRecord| in the user agent's |activeDocument|'s [=Document/prefetch records=]:
+  1. If |prefetchRecord|'s [=document state/initiator origin=] is [=same origin=] with |activeDocument|'s [=Document/origin=], then [=prefetch record/cancel and discard=] |prefetchRecord| given <var ignore>origin</var>.
```

Comment:
"initiator origin" is a property of a document state (as indicated by the syntax `[=document state/initiator origin=]`). It is not a property of prefetch records.

I think no such test is necessary here (although it might be in the implementation) since in the spec the prefetch records live on documents and so origin partitioning is automatic.

Reply:
I think I see what you mean here. If we [cancel and discard](https://wicg.github.io/nav-speculation/prefetch.html#prefetch-record-cancel-and-discardl) all prefetchRecords on a particular Document (in this case, the activeDocument where Clear-Site-Header header is coming from), then we know that their referring origin is same as Document's origin, since as you mentioned, there already is origin partitioning. Does that sound right to you?

Reply:
Yep, that sounds right to me!